### DISCUSSION TITLE
Compiling on Network Shares

### DISCUSSION THREAD

**Discussion by sliceofbytes** [glenn2223/vscode-live-sass-compiler#275] [Q&A] [answered]
Anyway to get this working on network shares, or ftp? Often times I just want to do a small edit on a remote server.

**Comment by glenn2223** [comment, marked as answer]
~It's recommended to use [VS Code Server](https://code.visualstudio.com/docs/remote/vscode-server) where this extension would be installed.~

Scratch that, though it is one option. VS Code has a whole section of documents on remote working you can see [here](https://code.visualstudio.com/docs/remote/remote-overview)
